packages/ilmomasiina-models/src/schema/signup/attributes.ts

```diff
@@ -92,3 +92,13 @@ export const dynamicSignupAttributes = Type.Object({
     description: "Whether the signup has been confirmed (saved).",
   }),
 });
+export const dynamicSignupAttributes = Type.Object({
+  ...publicDynamicSignupAttributes.properties,
```

Comment:
This needs doc comments, the doc comment above needs updating, and also pay attention to the code style (blank lines).